modules/age.nix

```diff
@@ -103,9 +117,33 @@ in
       message = "age.sshKeyPaths must be set.";
     }];
 
+    # Create a new directory full of secrets for symlinking (this helps
+    # ensure removed secrets are actually removed, or at least become
+    # invalid symlinks).
+    system.activationScripts.agenixMountSecrets = ''
+      _count="$(basename "$(readlink /run/secrets)" || echo 0)"
+      (( ++_count ))
+      echo "[agenix] symlinking new secrets generation $_count to /run/secrets..."
```

Comment:
@asymmetric Do you think I should also change the wording here to be something like `symlinking new secrets to /run/secrets (generation xyz)...`, so it matches with `removing old secrets (generation xyz)...`?

Reply:
Yes I think it would read better!